src/components/pages/Categories/Edit.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div>
+    <app-category-edit
+      :category-id="categoryId"
+      :current-category-name="currentCategoryName"
+      :access="access"
+      :disabled="disabled"
+      :error-message="errorMessage"
+      :done-message="doneMessage"
+      @updated-category="updatedCategory"
+      @handle-submit="handleSubmit"
+    />
+  </div>
+</template>
+
+<script>
+import CategoryEdit from '@Components/molecules/CategoryEdit/index.vue';
+
+export default {
+  components: {
+    appCategoryEdit: CategoryEdit,
+  },
+  computed: {
+    categoryId() {
+      let { id } = this.$route.params;
+      id = parseInt(id, 10);
+      return id;
+    },
```

Comment:
[imo]
わざわざ再代入しなくても良さそうですね！

```suggestion
    categoryId() {
      const { id } = this.$route.params;
      return parseInt(id, 10);
    },
```

Reply:
suggested　change見逃してて、別の書き方してしましました。

https://github.com/gizumo-education/gizumo_wiki_lesson/pull/165/commits/6eedee78799fe066e0d75e0b5b8a55b9da4a3d50